python/packages/autogen-ext/src/autogen_ext/agents/video_surfer/_video_surfer.py

```diff
@@ -13,6 +13,15 @@ class VideoSurferAgent(AssistantAgent):
 
     This agent utilizes various tools to extract information from the video, such as its length, screenshots at specific timestamps, and audio transcriptions. It processes these elements to provide detailed answers to user queries.
 
+    Available tools:
+    
+    - :func:`~autogen_ext.agents.video_surfer._action_space.extract_audio`
```

Comment:
We can export the tools as public module? `_action_space` implies private module.